### PR TITLE
Add Clever Cloud

### PR DESCRIPTION
**Pull request by davlgd**
**Similarweb rank:** 489,897
**Country Rank:** 49,332
**Category Rank:** 1,523

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

I used our [monochrome SVG logo](https://www.clever-cloud.com/app/themes/Starter/assets/img/brand-assets/mono_logo.svg) with our usual hex value color. 

I'm the product guy @ Clever Cloud.

### CODE REVIEW

**Comment by PeterShaggyNoble** [comment]
Thanks for the contribution, @davlgd 👍🏻 And the disclosure 🙂

But, seeing as this is so close to our 500k cut-off point and Similarweb only update their data once a month, we'll need to monitor this until their next update. That it's currently trending upwards, though, is a good sign.

Is there another metric you can provide in the meantime to illustrate the popularity of Clever Cloud?

**Comment by davlgd** [comment]
I'm not sure of what kind of metrics you need, but we're the major PaaS actor in the french market, with clients in 100+ countries, member of the [FrenTech DeepNum 20](https://presse.economie.gouv.fr/download/?n=257%20-%20DP_Laur%C3%A9ats_FrenchTech_DeepNum20_1%C3%A8re_%C3%A9dition-pdf&id=101801). We host websites for the french gov, often quote in the french digital strategy ([for example](https://www.numerique.gouv.fr/agenda/letat-dans-le-nuage-une-journee-dediee-au-cloud/) or [here](https://presse.economie.gouv.fr/06042023-cp-france-2030-vers-un-renforcement-de-loffre-cloud-de-confiance/))

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2708,6 +2708,11 @@
             "hex": "6C47FF",
             "source": "https://clerk.com"
         },
+        {
+            "title": "Clever Cloud",
+            "hex": "171C36",
+            "source": "https://www.clever-cloud.com/app/themes/Starter/assets/img/brand-assets/logo_on_dark.svg"
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:2714]:
```suggestion
            "source": "https://www.clever-cloud.com"
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
That's a good enough argument for me, @davlgd 👍🏻 Combined with the fact that it _is_ ranked over 500k _and_ trending upwards.

As you're an official representative of the brand, there's no need to review the icon treatment or colour choice here. However, there are a few excess points in your SVG; in all 3 corners of the bottom left triangle, above & below the right triangle, and below the left triangle. Could you see if they could be removed before we merge this?

And, for better future-proofing, I'd also suggest updating the `source` URL to be the website.

File: _data/simple-icons.json (hunk 1 above)
Comment (davlgd) [reply _data/simple-icons.json:2714]: Thanks, I thought the source URL should point to where the original logo is hosted. I'll commit the logo without the excessive points (sorry I missed them).

**Comment by PeterShaggyNoble** [comment]
Thanks for updating, @davlgd. However, there are still a number of excess points overlapping each other in all 3 corners of the bottom left triangle.

**Comment by davlgd** [comment]
Can you tell/show me where exactly ? Even with maximal zoom I don't see them

**Review by PeterShaggyNoble — APPROVED** [review]
That's weird! They were _definitely_ there on my last review, but they're gone now 🤷🏻‍♂️ The perils of staring at tiny icons all day, looking for the tiniest differences!

All looks good now - thanks again for you work on this one, @davlgd 👍🏻

**Pull request merged**